src/main/java/com/google/cloud/tools/managedcloudsdk/command/AsyncCommandWrapper.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.managedcloudsdk.command;
+
+import com.google.cloud.tools.managedcloudsdk.executors.SdkExecutorServiceFactory;
+import com.google.cloud.tools.managedcloudsdk.executors.SingleThreadExecutorServiceFactory;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.common.util.concurrent.ListeningExecutorService;
+import java.util.concurrent.Callable;
+
+/** Wrapper around {@link CommandExecutor} to asynchronous execution. */
+public class AsyncCommandWrapper {
```

Comment:
We've had some problems in the past with excess asynchronicity around commands.  I wonder if it should be left to the IDE to decide whether and how to execute a command asynchronously. E.g. in Eclipse we might want to use an Eclipse Job or bring up a progress monitor.

Reply:
Yeah, I think I'm going to take out the async stuff and let it all run synchronously.